drizzlepac/astrodrizzle.help

```diff
@@ -449,6 +445,15 @@ driz_sep_kernel : str {'square', 'point', 'gaussian', 'turbo', 'tophat', 'lanczo
         It leads to less resolution loss than other kernels, and typically
         results in reduced correlated noise in outputs.
 
+      * ``'tophat'``: (former option) this kernel is a circular "top hat" shape of 
+        width ``pixfrac``. It effects only output pixels within a radius of
+        ``pixfrac/2`` from the output position. This kernel has been removed
+        as it does not conserve flux. 
```

Comment:
1. I think this should be removed as an option altogether;
2. Other kernels have been found not to conserve flux but you've kept them (i.e., gaussian). The real reason it was removed it because it is buggy beyond repair (considering the effort top fix it and most likely very poor performance, time-wise, if fixed).

Reply:
That works for me.